repos/system_upgrade/el7toel8/actors/pythoninformuser/actor.py

```diff
@@ -0,0 +1,26 @@
+from leapp.actors import Actor
+from leapp.libraries.common import reporting
+from leapp.tags import ChecksPhaseTag
+from leapp.models import Report
+
+message = """
+Warning: In RHEL 8, there is no 'python' command. Python 3 is the primary Python version and Python 2 is available with limited support and limited set of packages.
+Read more here: FIXME
```

Comment:
```suggestion
Read more here: https://access.redhat.com/articles/4055961
```

Sadly, the article is not accessible to the public, nor (yet) to customers. It reuses information from the [Python in RHEL 8 blog post](https://developers.redhat.com/blog/2018/11/14/python-in-rhel-8/) and [RHEL 8 beta release notes](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8-beta/html-single/configuring_basic_system_settings/index#using-python3).